test/parallel/test-preload.js

```diff
@@ -8,9 +8,9 @@ var nodeBinary = process.argv[0];
 
 var preloadOption = function(preloads) {
   var option = '';
-  preloads.forEach(function(preload, index) {
-    // TODO: randomly pick -r or --require
-    option += '-r ' + preload + ' ';
+  preloads.forEach(function(preload) {
+    var optionString = ['-r', '--require'][Math.floor(Math.random()*2)];
```

Comment:
I don't like the random either but decided that I should stick to the `TODO` recommendation at least as a first pass.

I thought about running each test twice, once with `-r` and once with `--require`, but that misses the edge case of using both.

I think the best thing to do might be to put these tests back they way they were (using `-r` always) and just add two more tests: One that uses `--require` and one that uses both.